sockjs/transports/websocket.py

```diff
@@ -76,11 +76,14 @@ def process(self):
                 yield from ws.close()
                 return ws
 
-            server = asyncio.async(self.server(ws, self.session))
-            client = asyncio.async(self.client(ws, self.session))
+            server = asyncio.ensure_future(
```

Comment:
This change is backward incompatible, you need something like: https://github.com/KeepSafe/aiohttp/blob/master/aiohttp%2Fhelpers.py#L17-L20